### ISSUE TITLE
App name `Studio` is vague / too general – suggest `WordPress.com Studio` or `WordPress Studio`

### ISSUE THREAD

**Issue by haszari** [Automattic/studio#55]
### Quick summary

The app name is currently `Studio`. In my list of apps, it's not clear what it does (unless I look at the icon, and know about the WordPress logo). 

<img width="473" alt="Screenshot 2024-04-29 at 11 03 22 AM" src="https://github.com/Automattic/studio/assets/4167300/607d8118-c6b6-4759-92bd-369b6a081614">

### Steps to reproduce

1. Install.
2. Look at app in the context of other apps.

### What you expected to happen

The name should give clear clue about what the app does and doesn't do. `Studio` means a lot of things to many people (art, design, music, etc!), so I think it's important to signal clear scope – this thing is for prototyping WordPress websites.

### What actually happened

App is `Studio`, which is very general.

Note: individual users can work around by renaming the app (not a good thing to encourage).

### Impact

All

### Available workarounds?

Yes, easy to implement

**Comment by jeherve** [comment]
> WordPress Studio

I would recommend against that wording. It would make it sound like the app is an official WordPress product, when it's in fact an Automattic product.

Could it be "Studio by WordPress.com" instead, since we already use that wording in a few places?

**Comment by wojtekn** [comment]
> Could it be "Studio by WordPress.com" instead, since we already use that wording in a few places?

Correct, "Studio by WordPress.com" is the application's name, with "Studio" a short version.

**Comment by wojtekn** [comment]
Let's keep the app name short and simple, closing.

**Comment by haszari** [comment]
I see the app is now named `WordPress Studio` which is clearer - thanks! 

<img width="501" alt="Image" src="https://github.com/user-attachments/assets/de80ed77-48d1-4229-bd5c-519f6408406d" />

> Could it be "Studio by WordPress.com" instead, since we already use that wording in a few places?

> Correct, "Studio by WordPress.com" is the application's name, with "Studio" a short version.

I like this name even better – though in my install it's `WordPress Studio` for some reason.

**Comment by wojtekn** [comment]
@haszari I don't think we changed the app's name. Is it possible that you changed it on your side?

<img width="822" alt="Image" src="https://github.com/user-attachments/assets/00b4ed08-a8be-4676-8ffb-f8ec93b7b1c5" />

**Comment by haszari** [comment]
> I don't think we changed the app's name. Is it possible that you changed it on your side?

That's highly likely! 🤦🏻  😆
